discovery_engine_core/rust-toolchain.toml

```diff
@@ -1,3 +1,4 @@
 [toolchain]
 channel = "1.55.0"
 profile = "default"
+targets = ["aarch64-apple-ios", "x86_64-apple-ios"]
```

Comment:
This will force to install the targets in evrey job but we want to have them only in few of them so we can have an action or step to do it.